tables/zenodo-upload/README.md

```diff
@@ -187,16 +187,16 @@ We include `Kids_First_Biospecimen_DNA` and `Kids_First_Biospecimen_RNA` columns
 * For SNV data, we prioritize values from the consensus MAF file (`pbta-snv-consensus-mutation.maf.tsv.gz`) for inclusion in the following order: `HGVSp_Short`, `HGVSc`, and `Variant_Type`.
 When no change in the protein is noted in the `HGVSp_Short` value, we use the nucleotide change.
 * CNV alterations use the following notation from the `consensus_seg_annotated_*` files included in the data download: `<cytoband>-<status>-<copy_number>`.
-* We report the `FusionName` field from `pbta-fusion-putative-oncogenic.tsv` for fusions. 
+* We report the `FusionName` field from `pbta-fusion-putative-oncogenic.tsv` for fusions.
 When reciprocal fusions are detected, we only report one – whichever comes first when partner genes are sorted alphabetically.
 
 ### Caveats
 
 * Some `sample_id`-`composition` pairs map to multiple biospecimens of the same type (i.e., DNA and RNA).
-When this occurs, the individual biospecimen identifiers are separated with semi-colons in the relevant column, and `multiple_assays_within_type` is marked as `TRUE`. 
+When this occurs, the individual biospecimen identifiers are separated with semi-colons in the relevant column, and `multiple_assays_within_type` is marked as `TRUE`.
 The alterations for a gene are the union of alterations detected in biospecimens (i.e., they are detected in at least one biospecimen).
 * If multiple `cancer_type` or `broad_histology` values are associated with the same `sample_id`-`composition` pair, these values are semi-colon separated.
 * `germline_sex_estimate` is only available for `sample_id`-`composition` pairs that were assayed with WGS.
 * In two instances, multiple RNA biospecimens map to the sample `sample_id`-`composition` pair, and only one RNA biospecimen was assigned the `germline_sex_estimate` derived from the WGS data.
 We only included the non-missing values in these rows.
-The `sample_id` impacted are `7316-161` and `7316-85`.
+The `sample_id`s impacted are `7316-161` and `7316-85`.
```

Comment:
Yea, the rendered version is even more confusing than having this not agree, in my opinion. That's why I didn't add the s earlier.

Reply:
Do not disagree.. will clean up.